### PR TITLE
Add clause tokenize model

### PR DESCRIPTION
**Pull request by wannaphong**
Clause tokenize model trained from LST20 Corpus with CRF model.
LST20 Corpus from National Electronics and Computer Technology Center, Thailand. You can download dataset from [https://aiforthai.in.th/corpus.php](https://aiforthai.in.th/corpus.php).

Model train by Mr.Wannaphong Phatthiyaphaibun

Model License : CC-0

**Code**

- Model File : https://github.com/PyThaiNLP/pythainlp-corpus/releases/tag/lst20-cls-v0.1
- Train script : https://github.com/PyThaiNLP/pythainlp_notebook/blob/master/cls/lst20.ipynb
- Google Colab : [https://colab.research.google.com/drive/13sKMQN6FbI9YhyN3rVW9Ydg9bUjKk0JP?usp=sharing](https://colab.research.google.com/drive/13sKMQN6FbI9YhyN3rVW9Ydg9bUjKk0JP?usp=sharing)

**TODO**

- [x] document
- [x] test

### CODE REVIEW

**Comment by wannaphong** [comment]
# Model Card - LST20 CLS

**Model Details**
- Developer : Wannaphong Phatthiyaphaibun
- Model date : 2020-10-03
- Model version : 0.2
- CRF Model
- License : CC0

**Intended Use**
- Segmenting Thai text into clauses (smaller than sentence, bigger than word)
- Not suitable for other language or non-news domain.

**Factors**
- Based on known problems with thai natural Language processing.

**Metrics**
- Evaluation metrics include precision, recall and f1-score.

**Training Data**
LST20 Corpus Train set (news domain)

**Evaluation Data**
LST20 Corpus Test set (news domain)

**Quantitative Analyses**
```
              precision    recall  f1-score   support

       B_CLS       0.90      0.94      0.92     16111
       E_CLS       0.90      0.94      0.92     15947
       I_CLS       0.99      0.97      0.98    169565

   micro avg       0.97      0.97      0.97    201623
   macro avg       0.93      0.95      0.94    201623
weighted avg       0.97      0.97      0.97    201623
 samples avg       0.94      0.94      0.94    201623
```
**Ethical Considerations**
no ideas

**Caveats and Recommendations**

- You should segmente words before use.
- Thai text only

File: pythainlp/tokenize/crfcls.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+
+def _doc2features(doc, i):
+    # features from current word
+    curr_word = doc[i][0]
+    curr_pos = doc[i][1]
+    features = {
+        "word.word": curr_word,
+        "word.isspace": curr_word.isspace(),
+        "word.isdigit()": curr_word.isdigit(),
```
Comment (bact) [inline pythainlp/tokenize/crfcls.py]: ชื่อ feature แต่ละอันอาจจะลองดูให้สม่ำเสมอกันครับ
เช่นตอนนี้ isspace ไม่มี () แต่ isdigit() มี ()
prevword แต่ prepostag (ลืม v?)

หรือมีวิธีการตั้งชื่อที่สั้นกระชับ แต่ยังอ่านเข้าใจอยู่ไหม
เช่น

- word, word-1, word+1
- word, prev_word, next_word
- curr_isspace, prev_isspace, next_isspace
- -2.pos, -1.pos, 0.pos, +1.pos, +2.pos

เลือกเอาตามที่สะดวก และรู้สึกว่าเห็นแล้วก็เข้าใจได้ง่ายครับ

**Review by bact — CHANGES_REQUESTED** [review]
trying to make the feature names more consistent.

File: pythainlp/tokenize/crfcls.py (hunk 1 above)
Comment (wannaphong) [reply pythainlp/tokenize/crfcls.py]: @bact แก้ไขเรียบร้อยแล้วครับ

**Review by bact — APPROVED** [review]
